livechat/utils/ws_client.py

```diff
@@ -77,22 +87,35 @@ def send(self, request: dict, opcode=ABNF.OPCODE_TEXT) -> dict:
                 RtmResponse: RTM response structure (`request_id`, `action`,
                              `type`, `success` and `payload` properties)
         '''
-        response_timeout = self.response_timeout
         request_id = str(random.randint(1, 9999999999))
         request.update({'request_id': request_id})
         request_json = json.dumps(request, indent=4)
         logger.info(f'\nREQUEST:\n{request_json}')
+
         if not self.sock or self.sock.send(request_json, opcode) == 0:
             raise WebSocketConnectionClosedException(
                 'Connection is already closed.')
-        while not (response := next(
-            (item
-             for item in self.messages if item.get('request_id') == request_id
-             and item.get('type') == 'response'),
-                None)) and response_timeout > 0:
-            sleep(0.2)
-            response_timeout -= 0.2
-        logger.info(f'\nRESPONSE:\n{json.dumps(response, indent=4)}')
+        
+        stop = False
```

Comment:
please try with threading.Event() which could be safer choice for threading flag